sklearn/manifold/_mds.py

```diff
@@ -272,6 +274,72 @@ def smacof(dissimilarities, metric=True, n_components=2, init=None, n_init=8,
         return best_pos, best_stress
 
 
+def svd_scaler(dissimilarities, n_components=2):
+    """
+    Computes multidimensional scaling using SVD algorithm
+
+    Parameters
+    ----------
+    dissimilarities : ndarray, shape (n_samples, n_samples)
+        Pairwise dissimilarities between the points. Must be euclidean.
```

Comment:
Here and in several other parts

> Must be euclidean

I may be mistaken but the term "Euclidean" when applied to metrics/distance does not denote a property but instead refers to **the** euclidean distance metric (the L2 norm of vector difference). 

So what you probably mean hare and in other comments is that the dissimilarity must be a **metric**, that is, it should ensure the 4 conditions below:

 - positivity
 - symmetry
 - distinguishability
 - triangular inequality

Ref: M.M. Deza and E. Deza. Encyclopedia of Distances. Vol. 2006. 2009, p. 590. arXiv:0505065

Or simply: https://en.wikipedia.org/wiki/Metric_(mathematics)#Definition